cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/inmemory/InMemoryMapReduceContextBuilder.java

```diff
@@ -86,7 +88,13 @@ private Injector createInMemoryModules() {
       new LoggingModules().getInMemoryModules(),
       new StreamAdminModules().getInMemoryModules(),
       new ExploreClientModule(),
-      new NotificationFeedServiceRuntimeModule().getInMemoryModules()
+      new NotificationFeedServiceRuntimeModule().getInMemoryModules(),
+      new AbstractModule() {
+        @Override
+        protected void configure() {
+          bind(Store.class).to(DefaultStore.class);
```

Comment:
Do we need it here? I thought only the program runner uses Store and program runner is not created by this injector, but rather by the one in AppFabric for in-memory (which already has Store binded) and in `AbstractProgramTwillRunnable` for distributed mode.